bitnami/metallb/templates/controller/servicemonitor.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.controller.prometheus.serviceMonitor.enabled }}
```

Comment:
Same, we usually separate each object in a different file.

Reply:
Will fix that